config/environments/development.rb

```diff
@@ -3,6 +3,12 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  # Do not dump schema automatically after migrations
+  config.active_record.dump_schema_after_migration = false
+
+  # Allow all hosts to connect
+  config.hosts = nil
```

Comment:
Disabling this will make us vulnerable to DNS rebinding attacks as described [here](https://blog.saeloun.com/2019/10/31/rails-6-adds-guard-against-dns-rebinding-attacks.html) so we shouldn't make this the default setup. For your setup could you instead append an allowed host configured via an environment variable?

Reply:
Got it.